### PR TITLE
Added noexcept to many functions, trying to solve #131

### PR DESCRIPTION
**Pull request by Laiot**

### CODE REVIEW

File: include/NovelRT/Graphics/RenderObject.h [hunk 1]
```diff
@@ -11,10 +11,10 @@ namespace NovelRT::Graphics {
   class RenderObject : public WorldObject {
   protected:
     virtual void drawObject() = 0;
-    virtual void configureObjectBuffers();
-    static GLuint generateStandardBuffer();
-    Maths::GeoMatrix4x4<float> generateViewData();
-    Maths::GeoMatrix4x4<float> generateCameraBlock();
+    virtual void configureObjectBuffers() ;
```
Comment (RubyNova) [inline include/NovelRT/Graphics/RenderObject.h:14]: Uhhh why is there a space here?

File: src/NovelRT/Maths/GeoBounds.cpp [hunk 2]
```diff
@@ -12,11 +12,9 @@ namespace NovelRT::Maths {
     auto corner0 = getCornerInWorldSpace(0);
     auto corner2 = getCornerInWorldSpace(2);
 
-    if (point.getX() >= corner0.getX() && point.getX() <= corner2.getX() && point.getY() <= corner2.getY()
-      && point.getY() >= corner0.getY())
-      return true;
+      return point.getX() >= corner0.getX() && point.getX() <= corner2.getX() && point.getY() <= corner2.getY()
+             && point.getY() >= corner0.getY();
```
Comment (RubyNova) [inline src/NovelRT/Maths/GeoBounds.cpp:16]: I assume you ran the unit tests after changing this? Or are there no unit tests for `GeoBounds` yet? (I've been busy getting the tests caught up, sorry >_<).

Comment (capnkenny) [reply src/NovelRT/Maths/GeoBounds.cpp:16]: @RubyNova no tests for `GeoBounds` at the moment

Comment (RubyNova) [reply src/NovelRT/Maths/GeoBounds.cpp:16]: OK, then we really ought to add some.

We can't have changes like this without the tests to make it airtight, really.

Maybe @Laiot would like to add some?

**Comment by RubyNova** [comment]
PR is dead. Closing.

**Pull request closed without merging**